test/e2e/clusterclass_changes.go

```diff
@@ -912,8 +910,6 @@ func cloneRef(ctx context.Context, cl client.Client, ref *corev1.ObjectReference
 	template.SetResourceVersion("")
 	template.SetOwnerReferences(nil)
 	Expect(cl.Create(ctx, template)).To(Succeed())
```

Comment:
How is the ClusterClass now using the new templates?

Reply:
Fixed, PTAL